ui/src/components/AT_HEADER.css

```diff
@@ -30,19 +36,36 @@
   font: normal normal normal 32px/33px Abolition;
 }
 
-#atc-header ul {
-  float: right;
+#atc-header h2 {
+  padding-top: 40px;
+  color: #fff;
+  font: normal normal normal 25px/28px Abolition;
+  filter: drop-shadow(1px 1px 1px rgba(0, 0, 0, 0.877));
+}
+
+.ant-menu {
+  width: 100%;
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
+  display: block;
+}
+
+.ant-menu-item {
   list-style: none;
+  float: right !important;
```

Comment:
does it work in smaller screens?

Reply:
Good call. On smaller screens, it doesn't work. I'll make a new commit once it's fixed.